integration/kafka/src/main/java/alluxio/kafka/connect/AlluxioSinkConnector.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the “License”). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.kafka.connect;
+
+import org.apache.kafka.connect.connector.Connector;
+import org.apache.kafka.connect.connector.Task;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This class is a Kafka Connect Connector implementation that exports data from Kafka to Alluxio.
+ */
+public final class AlluxioSinkConnector extends Connector {
+
+  private Map<String, String> mConfigProperties;
+
+  @Override
+  public String version() {
+    return AlluxioSinkConnectorConfig.ALLUXIO_CONNECTOR_VERSION;
```

Comment:
In the HDFS and JDBC's connector implementation, we used the project version as the connector version. For example, if in the pom.xml, the project version 3.0.0, then the connector version will be 3.0.0. Using project version instead of providing version in the connector configuration, we have less chance to provide the wrong value.